### PR TITLE
[libxkbcommon] Update to 1.3.1. Fixes JB#55591

### PR DESCRIPTION
**Pull request by Tomin1**
Convert to submodule format. Update it to 1.3.1. Update spec. Use
summary and description texts from Fedora.

Add simple build system patch to skip building of utilities.

### CODE REVIEW

File: rpm/libxkbcommon.spec [hunk 1]
```diff
@@ -1,58 +1,58 @@
...
 Source0:    libxkbcommon-%{version}.tar.bz2
+Patch0:     0001-Add-enable-utils-meson-option.patch
 Requires(post): /sbin/ldconfig
 Requires(postun): /sbin/ldconfig
+BuildRequires:  meson
 BuildRequires:  byacc
-BuildRequires:  libtool
 BuildRequires:  flex
 BuildRequires:  bison
+BuildRequires:  xkeyboard-config-devel
```
Comment (mlehtima) [inline rpm/libxkbcommon.spec]: Would using pkgconfig dependency look better "pkgconfig(xkeyboard-config)"?

**Comment by saukko** [comment]
Would split the subtree removal to separate commit so that one can actually see what was changd in addition :)

File: rpm/libxkbcommon.spec (hunk 1 above)
Comment (Tomin1) [reply rpm/libxkbcommon.spec]: Yes, very much so. :+1: 

Then again we could actually skip that dependency as all it seems to need from that is xkb_base path (seems to be /usr/share/X11/xkb). If we trust that's constant for us, then there is no need to look it up and we can set it with build option instead. Probably doesn't matter very much wrt. build time etc.

**Comment by Tomin1** [comment]
> Would split the subtree removal to separate commit so that one can actually see what was changd in addition :)

It's now split.

**Review by saukko — APPROVED** [review]
LGTM from packaging point of view, didn't test on device but looks ok in general.

**Review by mlehtima — APPROVED** [review]

**Review by mkosola — APPROVED** [review]

**Pull request merged**